scico/linop/optics.py

```diff
@@ -91,15 +111,19 @@ def __init__(
     ):
         r"""
         Args:
-            input_shape: Shape of input array. Can be a tuple of length
-               1 or 2.
+            input_shape: Shape of input array as a tuple of length
+               1 or 2, corresponding to :math:`(N_x,)` or
+               :math:`(N_x, N_y)`.
             dx: Sampling interval at source plane. If a float and
-               `len(input_shape)==2` the same sampling interval is
-               applied to both dimensions. If `dx` is a tuple, must
-               have same length as `input_shape`.
-            k0: Illumination wavenumber; 2π/wavelength.
-            z: Propagation distance.
-            pad_factor: Amount of padding to apply in DFT step.
+               `len(input_shape)==2` the same sampling interval is applied
+               to both dimensions. If `dx` is a tuple, it must have same
+               length as `input_shape`, and corresponds to either
+               :math:`(\Delta x,)` or :math:`(\Delta x, \Delta y)`.
+            k0: Illumination wavenumber, :math:`k_0`, corresponding to
+               :math:`2 \pi` / wavelength.
+            z: Propagation distance, :math:`z`.
+            pad_factor: The padded input shape is the input shape
```

Comment:
@lukepfister: Can you suggest some guidance to the user on selection of the padding factor?

Reply:
That's a complicated subject;  it's the same as the choice of padding for DFT-based linear convolution but has extra physical interpretation.  Would probably just leave a pointer to Chapter 5 of voelz.

Reply:
Better than nothing, I guess, but I was hoping to provide some guidance that didn't require the user to first do enough reading to be able to write their own implementation!

Reply:
🤷‍♂️ I think it is too complicated to get into the docs. Unlike a standard analysis of convolution in a signal processing class, the side lengths have physical meaning. Sometimes the size of your domain is just the size of your domain and any spatial aliasing is really indicative that you haven’t modeled the transmission channel properly (not accounting for reflections, etc) 

there is a case to be made that those options should be removed entirely and the user should set up the domain side lengths to be larger.